include/freerdp/channels/rdpdr.h

```diff
@@ -126,12 +126,14 @@ enum IRP_MN
 #define FILE_OVERWRITTEN 0x00000003
 
 /* DR_CORE_CLIENT_ANNOUNCE_RSP.VersionMinor */
+#define RDPDR_MAJOR_RDP_VERSION 1
```

Comment:
Should make this an `enum` too like `RDPDR_MINOR_RDP_VERSION`

Reply:
can do that as soon as we have a second value to check for.